### PR TITLE
new parameters in data transfer file

### PR DESCRIPTION
**Pull request by javiteri95**
new parameters in data transfer file

### CODE REVIEW

File: electron_app/src/database/DBEexporter.js [hunk 1]
```diff
@@ -844,10 +848,29 @@ const exportEncryptDatabaseToFile = async ({
     ? accountObj.recipientId.split('@')
     : myAccount.recipientId.split('@');
   const accountId = accountObj ? accountObj.id : myAccount.id;
+  const signature = accountObj
```
Comment (danieltigse) [inline electron_app/src/database/DBEexporter.js:851]: `accountObj && accountObj.signature ? accountObj.signature : myAccount.signature`

File: email_mailbox/src/components/ManualSyncProcessPopup.js [hunk 2]
```diff
@@ -147,6 +147,7 @@ class ManualSyncProcessPopup extends Component {
   checkDataStatus = async () => {
     const { status, body } = await getDataReady(myAccount.recipientId);
     if (status === DATA_READY_STATUS) {
+      //here must exist a clear timeout
```
Comment (danieltigse) [inline email_mailbox/src/components/ManualSyncProcessPopup.js:150]: why don't you added?

File: electron_app/src/database/DBEexporter.js [hunk 3]
```diff
@@ -844,10 +848,29 @@ const exportEncryptDatabaseToFile = async ({
...
+    ? accountObj.signature !== undefined
+      ? accountObj.signature
+      : myAccount.signature
+    : myAccount.signature;
+  const hasCriptextFooter = accountObj
+    ? accountObj.signFooter !== undefined
```
Comment (Hirobreak) [inline electron_app/src/database/DBEexporter.js:857]: Same as above

File: email_mailbox/src/components/ManualSyncProcessPopup.js (hunk 2 above)
Comment (javiteri95) [reply email_mailbox/src/components/ManualSyncProcessPopup.js:150]: Not entirely sure, i will check again

**Pull request merged**